spec/services/preserved_object_handler_spec.rb

```diff
@@ -67,9 +67,78 @@
   end
 
   describe '#check_existence' do
-    # FIXME: move the actual specs to a separate file?
```

Comment:
I believe we would like the check_existence specs to live in a separate file, as indicated by the comment?